### PR TITLE
fix source URL for KEGG.db extension in R-bundle-Bioconductor 3.13 easyconfig (since it's a deprecated library now)

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb [hunk 1]
```diff
@@ -1047,9 +1050,6 @@ exts_list = [
     ('GenABEL', '1.8-0', {
         'checksums': ['434fc806b918738ec67171a37a47099d1706332819317e368fe0ccb8f149a469'],
     }),
-    ('JASPAR2020', '0.99.10', {
```
Comment (boegel) [inline easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb:1050]: @SIB-software It seems like this was removed unintentionally?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb]: Ahh! It was not based on the last file version.
Should be fixed now

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3119.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/01f8a59981c1fd15e541f3490abc7055 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**